daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SValue.scala

```diff
@@ -187,6 +195,10 @@ object SValue {
   }
 
   final case class SAny(ty: Type, value: SValue) extends SValue
+  final case class SAnyException(ty: Type, messageFunction: SExpr, value: SValue) extends SValue
+
+  // A value of one of the builtin exception types: GeneralError, ArithmeticError, ContractError
```

Comment:
I'm loathe to add any more structure here at the moment, because I am not convinced that this speedy-value tagging is required at all. It's not used by speedy execution in any place. And speedy-values should never escape the execution.

We can talk about this more later.

Reply:
fair enough, then let’s add some todos/or issues. If we can kill it completely that’s also fine. I just want to get rid of the string tag.